xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-tree/xwiki-platform-index-tree-macro/src/main/resources/XWiki/DocumentTreeMacros.xml

```diff
@@ -787,7 +787,7 @@
 #macro (addPaginationNode $parentId $offset $totalCount $siblings)
   #set ($discard = $siblings.add({
     'id': "pagination:$parentId",
-    'text': "$!mathtool.sub($totalCount, $offset) more ...",
+    'text': "$!mathtool.sub($totalCount, $offset) $services.localization.render('index.documentTree.more')",
     'icon': 'fa fa-eye',
```

Comment:
I would rather add a parametrized key, which would allow the translator to also change the order between the displayed number and the text associated to this number, as the order between the 2 depends on the language. Take  the example of the romanian language, you'd say "Încă 12..." while in english you would say "12 more...". This type of translation does not allow the romanian translation to be done properly, as the formula must absolutely start with the number.